configure.ac

```diff
@@ -1444,13 +1444,21 @@ if test -n "$LIBAIO"; then
 fi
 
 BUILD_LIBURING=no
+AC_ARG_ENABLE([linux-io_uring],
+              AC_HELP_STRING([--disable-linux-io_uring],
+                             [Disable io-uring in POSIX xlator.]))
 AC_CHECK_HEADERS([liburing.h])
 AC_CHECK_LIB([uring],[io_uring_queue_init],[LIBURING="-luring"])
```

Comment:
This should only be done if `linux-io_uring` is enabled. Additionally, we do really need both checks ? isn't correct to assume that if the header is present, the library should also be present ?

Reply:
- I will move it to inside the if conditional.
- Yes it is better to have both checks since distributions typically seperate out the headers into a separate package.

On Fedora 32:
```
[~]$ rpm -ql liburing-0.7-2.fc32.x86_64
/usr/lib/.build-id
/usr/lib/.build-id/bb
/usr/lib/.build-id/bb/d3cffb99015dad95193529623fdb4577eae271
/usr/lib64/liburing.so.1
/usr/lib64/liburing.so.1.0.7
/usr/share/licenses/liburing
/usr/share/licenses/liburing/COPYING
```

```
[~]$ rpm -ql liburing-devel-0.7-2.fc32.x86_64
/usr/include/liburing
/usr/include/liburing.h
/usr/include/liburing/barrier.h
/usr/include/liburing/compat.h
/usr/include/liburing/io_uring.h
/usr/lib64/liburing.so
/usr/lib64/pkgconfig/liburing.pc
/usr/share/man/man2/io_uring_enter.2.gz
/usr/share/man/man2/io_uring_register.2.gz
/usr/share/man/man2/io_uring_setup.2.gz
```

Reply:
Actually, I am not able to remove the library rpm when the devel rpm is installed, so I think you are right that having headers installed implies presence of the libs too.

Reply:
But `-devel` packages normally (always ?) depend on the library, so if the headers are there, the library should also be there.